CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ IMPROVEMENTS:
    an idempotent operation. [GH-1903]
  * credential/approle: At least one constraint is required to be enabled while
    creating and updating a role [GH-1882]
+ * secret/cassandra: Added consistency level for use with roles [GH-1931]
```

Comment:
[Optional] Can we update the changelog after the PR gets merged as part of a separate commit? This is to avoid merge conflicts with the CL.